### PR TITLE
Theme Designer: creating child components and basic layout

### PR DESCRIPTION
**Pull request by emmayjiang**
Creating a layout and adding child components for the theme designer, which is originally a stub.
image

### CODE REVIEW

File: packages/react-components/theme-designer/src/ThemeDesigner.tsx [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
+import { makeStyles } from "@griffel/react";
+
+import { Nav } from './components/Nav'
+import { Sidebar } from './components/Sidebar'
+import { Body } from './components/Body'
+
+const borderColor: string = "#D1D1D1";
+
+const useLayouts = makeStyles({
+  container: {
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/ThemeDesigner.tsx:12]: top level className is typically 'root'

File: packages/react-components/theme-designer/src/ThemeDesigner.tsx [hunk 2]
```diff
@@ -1,9 +1,58 @@
...
  * ThemeDesigner component - TODO: add more docs
  */
 export const ThemeDesigner: React.FC<ThemeDesignerProps> = props => {
-  return <div>hello</div>;
+  const layouts = useLayouts();
+  return (
+    <div className={layouts.container}>
+      <div className={layouts.nav}><Nav /></div>
+      <div className={layouts.sidebar}><Sidebar /></div>
+      <div className={layouts.body}><Body /></div>
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/ThemeDesigner.tsx:55]: we can move the classname onto the component, and reduce the amount of DOM on the page. Just make sure to mix in props.className with the className of the control

```suggestion
      <Nav className={layouts.nav} />
      <Sidebar className={layouts.sidebar} />
      <Body className={layouts.body} />
```

File: packages/react-components/theme-designer/src/components/Body/Body.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { makeStyles } from "@griffel/react";
+
+export interface ComponentProps {}
+
+const useStyles = makeStyles({
+  body: {
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Body/Body.tsx:7]:
```suggestion
  root: {
```

File: packages/react-components/theme-designer/src/components/Nav/Nav.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { makeStyles } from "@griffel/react";
+
+export interface ComponentProps {}
+
+const useStyles = makeStyles({
+  nav: {
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Nav/Nav.tsx:7]:
```suggestion
  root: {
```

File: packages/react-components/theme-designer/src/components/Nav/Nav.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  }
+});
+
+export const Nav: React.FC<ComponentProps> = props => {
+  const styles = useStyles();
+  return (
+    <div className={styles.nav}>Nav</div>
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Nav/Nav.tsx:14]:
```suggestion
    <div className={styles.root}>Nav</div>
```

File: packages/react-components/theme-designer/src/components/Body/Body.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  }
+});
+
+export const Body: React.FC<ComponentProps> = props => {
+  const styles = useStyles();
+  return (
+    <div className={styles.body}>Body</div>
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Body/Body.tsx:14]:
```suggestion
    <div className={styles.root}>Body</div>
```

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/7c773e3f3f4a8991fee6af46277504378dbdf197'>7c773e3f3f4a8991fee6af46277504378dbdf197</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/theme-designer/src/ThemeDesigner.tsx [hunk 7]
```diff
@@ -1,9 +1,58 @@
...
 export const ThemeDesigner: React.FC<ThemeDesignerProps> = props => {
-  return <div>hello</div>;
+  const layouts = useLayouts();
+  return (
+    <div className={layouts.root}>
+      <Nav className={layouts.nav} />
+      <Sidebar className={layouts.sidebar} />
+      <Body className={layouts.body} />
+    </div>
+  );
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/ThemeDesigner.tsx:57]:
```suggestion
  const styles = useLayouts();
  return (
    <div className={styles.root}>
      <Nav className={styles.nav} />
      <Sidebar className={styles.sidebar} />
      <Body className={styles.body} />
    </div>
  );
```

**Comment by emmayjiang** [comment]
https://github.com/microsoft/fluentui/pull/23158

**Pull request closed without merging**